var/spack/repos/builtin/packages/gaussian-view/package.py

```diff
@@ -18,15 +18,26 @@ class GaussianView(Package):
     homepage = "https://gaussian.com/gaussview6/"
     manual_download = True
 
-    version('6016',
+    version('6.1.6',
+            sha256='c9824fd0372c27425b53de350f3a83b27de75ca694219b3ef18cd7d92937db6c',
+            extension='tbz')
+
+    version('6.0.16',
             '5dd6a8df8c81763e43a308b3a18d2d3b825d3597e9628dcf43e563d1867b9638',
             extension='tbz')
 
-    depends_on('gaussian@16-B.01', type='run')
+    depends_on('gaussian@16-B.01', type='run', when='@:6.0.99')
+    depends_on('gaussian@16-C.01', type='run', when='@6.1:')
 
     def url_for_version(self, version):
-        return "file://{0}/gv-{1}-Linux-x86_64.tbz".format(os.getcwd(),
-                                                           version)
+        if '@:6.0.99' in self.spec:
```

Comment:
What do say if just force everyone to use one of the two formats? If you want to install it using Spack the file has to be called either `gv-{1}-Linux-x86_64.tbz` or `gv{1}-linux-x86_64.tbz`. You pick the one you like most 😄

Reply:
ok, will do. I just didn't want to break current procedure (on the other hand I probably did just that by building Gaussian).